packages/bpk-component-snippet/src/BpkSnippet.tsx

```diff
@@ -143,19 +159,35 @@ const BpkSnippet = ({
       }
     >
       {headline && (
-        <div className={getClassName('bpk-snippet--headline')}>
-          <BpkText textStyle={headlineStyle}>{headline}</BpkText>
-        </div>
+        <BpkBreakpoint query={BREAKPOINTS.MOBILE}>
+          {(isMobile: boolean) => (
+            <div className={getClassName('bpk-snippet--headline')}>
+              <BpkText
+                textStyle={isMobile ? HEADLINE_STYLE.heading4 : headlineStyle}
+              >
+                {headline}
+              </BpkText>
+            </div>
+          )}
+        </BpkBreakpoint>
       )}
       {subheading && (
         <div className={getClassName('bpk-snippet--subheading')}>
           <BpkText textStyle={TEXT_STYLES.subheading}>{subheading}</BpkText>
         </div>
       )}
       {bodyText && (
-        <div className={getClassName('bpk-snippet--bodyText')}>
-          <BpkText textStyle={bodyStyle}>{bodyText}</BpkText>
-        </div>
+        <BpkBreakpoint query={BREAKPOINTS.MOBILE}>
+          {(isMobile: boolean) => (
+            <div className={getClassName('bpk-snippet--bodyText')}>
+              <BpkText
+                textStyle={isMobile ? BODY_STYLE.bodyDefault : bodyStyle}
+              >
+                {bodyText}
+              </BpkText>
+            </div>
+          )}
+        </BpkBreakpoint>
```

Comment:
Rather than having two different setups of BpkBreakpoint I would just wrap the whole code in BpkBreakpoint and then use the same variable for controlling the code :)

Reply:
👍👍 I should have taken that into account, have updated!